### PR TITLE
UMD + Linting

### PR DESCRIPTION
**Pull request by stephane-tessier**
Hi,

Thanks for the great job! :+1: 

I suggest turning this module into a Universal Module, so that it can be included more easily into some projects.
Besides, some linting let me see that you declare variables that you don't use ;)

### CODE REVIEW

**Comment by nico3333fr** [comment]
Hi,

thanks :)

yes, this script needs a lot of cleanup, you've made a good start here :) (it is one of the oldest I've written)

I hope to perform a full and clean rewrite soon on it, as @ryuran did for accordion script https://github.com/nico3333fr/jquery-accessible-accordion-aria 

Quick question and advice needed: I've noticed you added .eslintrc.json and @ryuran used .jshintrc .
 
ShouId I put both on the repo ?

Anyway, thanks a lot for your help 👍

**Comment by ryuran** [comment]
@nico3333fr jshint will be disappear for eslint.
https://www.slant.co/versus/8627/8628/~jshint_vs_eslint

Furthermore jshint do not control coding style and need jscs to complete this missing.

But jscs has merged with eslint. http://jscs.info/
So eslint covers all the scope for linting and code styling control.

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.{js,scss,css,html,hbs,twig,json}]
+indent_style = space
+indent_size  = 4
```
Comment (ryuran) [inline .editorconfig]: Missing new line

**Review by ryuran — CHANGES_REQUESTED** [review]

File: .editorconfig (hunk 1 above)
Comment (stephane-tessier) [reply .editorconfig]: fixed, thanks!

**Comment by nico3333fr** [comment]
@ryuran so it is better to use settings provided in this example ? I guess I will have to update them in accordion.

Both @ryuran and @stephane-tessier : Thanks a lot for your help 👍

**Comment by stephane-tessier** [comment]
:smile:

**Comment by ryuran** [comment]
@nico3333fr This one is set to 4 space indent size.
For accordion is set to 2.
You need to make some choice about the coding style.
It can be a good idea to put the eslint config on a specific package http://eslint.org/docs/user-guide/configuring#using-a-shareable-configuration-package and document your coding style here in this repo.

Each project of your collection will refer to this external settings and will respect same rules.

**Pull request merged**